src/components/Global/ImageGrid.vue

```diff
@@ -116,15 +108,35 @@ watch(() => props.images, () => {
             class="image-text-overlay"
           >{{ image.operationIndex }}</span>
         </v-img>
-        <v-skeleton-loader
-          v-else
-          type="card"
-          color="var(--secondary-background)"
-          bg-color="var(--primary-background)"
-        />
-      </v-col>
-    </template>
-    <template v-else>
+        <div
+          v-if="props.enableImageCards"
+          class="d-flex flex-row ga-2 align-center mt-2"
+        >
+          <p class="text-subtitle-2 mr-auto prevent-select single-line-text">
+            {{ image.target_name }}
```

Comment:
Datalab output images don't have a target_name field (or a filter field for that matter). Maybe you can put something else here? Maybe what operation it came from (i.e. Median, Subtraction, RGB) or something else relevant.